extensions/catppuccin/package.json

```
@@ -85,6 +85,32 @@
           "title": "Large"
         }
       ]
+    },
+    {
+      "name": "flavor",
+      "type": "dropdown",
+      "required": false,
+      "title": "Catppuccin Flavor",
+      "description": "Select your favorite Catppuccin flavor to default to",
+      "default": "mocha",
+      "data": [
+        {
+          "title": "Mocha",
+          "value": "mocha"
+        },
+        {
+          "title": "Macchiato",
+          "value": "macchiato"
+        },
+        {
+          "title": "Frappe",
```

```suggestion
          "title": "Frappé",
```
This should technically be the name of frappe if Raycast handles the accents fine.

---

raycast handles the accents no issue, updated